### PR TITLE
MAYA-108545: update error message string.

### PR DESCRIPTION
**Pull request by HamedSabri-adsk**
Minor error message string change.

### CODE REVIEW

File: lib/mayaUsd/ufe/UsdTransform3dHandler.cpp [hunk 1]
```diff
@@ -50,7 +50,10 @@ Ufe::Transform3d::Ptr UsdTransform3dHandler::transform3d(const Ufe::SceneItem::P
     // not allowed.
     // https://graphics.pixar.com/usd/docs/api/_usd__page__scenegraph_instancing.html#Usd_ScenegraphInstancing_InstanceProxies
     if (usdItem->prim().IsInstanceProxy()) {
-        MGlobal::displayError("Authoring to an instance proxy is not allowed.");
+        MGlobal::displayError(
+            MString("Authoring to the descendant of an instance [")
+            + MString(usdItem->prim().GetName().GetString().c_str()) + MString("] is not allowed.")
```
Comment (ppt-adsk) [inline lib/mayaUsd/ufe/UsdTransform3dHandler.cpp:55]: Wow. item get prim get name get string get c string.

**Review by ppt-adsk — APPROVED** [review]

File: lib/mayaUsd/ufe/UsdTransform3dHandler.cpp (hunk 1 above)
Comment (HamedSabri-adsk) [reply lib/mayaUsd/ufe/UsdTransform3dHandler.cpp:55]: Yup! Tell me about it... :)

Comment (mattyjams) [reply lib/mayaUsd/ufe/UsdTransform3dHandler.cpp:55]: `TfToken::GetText()` could at least trim down the "get string get c string" part:
```C++
usdItem->prim().GetName().GetText()
```

Comment (HamedSabri-adsk) [reply lib/mayaUsd/ufe/UsdTransform3dHandler.cpp:55]: Ugh Good point... I have to refactor this code anyways later. Will make the change when it happens.

**Pull request merged**